Source/Core/Transforms.js

```diff
@@ -523,7 +523,7 @@ define([
      * indicates that the preload has completed.
      *
      * @param {TimeInterval} timeInterval The interval to preload.
-     * @returns {Promise} A promise that, when resolved, indicates that the preload has completed
+     * @returns {Promise.<undefined>} A promise that, when resolved, indicates that the preload has completed
```

Comment:
Similar to above, the resolved value is not useful.  From examining the code, it's an array of array of undefined, but that shouldn't be documented.